examples/diff/index.html

```diff
@@ -36,43 +37,55 @@
 <!-- Diffing UI Extension -->
 <script>
 
-  function DiffExtension(extensionsApi) {
+  function diffExtension(extensionsApi) {
 
-    extensionsApi.window.updateHeight();
+    extensionsApi.window.startAutoResizer();
 
     // DOM elements.
     var draftTextEl = document.querySelector('.draft-text');
     var diffTextEl = document.querySelector('.diff-text');
 
+    // Disable the input as we need to wait for an async call to load the published entry.
+    draftTextEl.disabled = true;
+
+    var entry = extensionsApi.entry;
+    var field = extensionsApi.field;
+
     // Currently published version of entry.
     var publishedVersion = extensionsApi.entry.getSys().publishedVersion;
+    var publishedValue;
 
     // Current locale
     var fieldLocale = extensionsApi.field.locale;
     var fieldId = extensionsApi.field.id;
 
-    var publishedValue;
-
-    // Populate the input field with the value of the entry.field.
-    draftTextEl.value = extensionsApi.entry.fields[extensionsApi.field.id].getValue();
+    // Callbacks for changes on the field or the entry.sys object.
+    // Detach functions could be used to clean up the callbacks.
+    var detachValueChanged = field.onValueChanged(onValueChanged);
```

Comment:
You could invoke the detach methods when this iframe is unloaded. So something like: 
```js
window.addEventListener('beforeunload', function () {
  detachValueChanged();
  detachSysChanged();
});
```